docs/README.txt

```diff
@@ -1,7 +1,34 @@
+If you are reading Tahoe-LAFS documentation
+-------------------------------------------
 
-Note: http://tahoe-lafs.readthedocs.io/en/latest/ is the preferred place to
-read this documentation (GitHub doesn't render cross-document links or
-images). If you're reading this on https://github.com/tahoe-lafs/tahoe-lafs ,
-or from a checked-out source tree, then either run `tox -e docs` and open
-_build/html/index.html in your browser, or view the pre-rendered trunk copy
-at http://tahoe-lafs.readthedocs.io/en/latest/
+If you are reading Tahoe-LAFS documentation at a code hosting site or
+from a checked-out source tree, the preferred place to view the docs
+is http://tahoe-lafs.readthedocs.io/en/latest/.  Code-hosting sites do
+not render cross-document links or images correctly.
+
+
+If you are writing Tahoe-LAFS documentation
+-------------------------------------------
+
+To edit Tahoe-LAFS docs, you will need a checked-out source tree.  You
+can edit the `.rst` files in this directory using a text editor, and
+then generate HTML output using Sphinx, a program that can produce its
+output in HTML and other formats.
+
+The files with `.rst` extension use reStructuredText markup format,
+which is the format Sphinx natively handles.  To learn more about
+Sphinx, and for a friendly primer on reStructuredText, please see
+Sphinx project's documentation, available at:
+
+https://www.sphinx-doc.org/
+
+If you have tox installed, you can run `tox -e docs` and then open the
+resulting docs/_build/html/index.html in your web browser.
+
+If you have Sphinx and Make installed, you can also run `make html`
+within docs directory.  You may also need to install some additional
+Python packages, such as setuptools and recommonmark.
```

Comment:
`setuptools` and `recommonmark`.

Reply:
We shouldn't even have lists of dependencies in text files.  Instead, we should just direct people to run the tox command or, at worst, something like `pip install tahoe-lafs[docs]`.

Reply:
Probably a minor thing, but installing Sphinx plugins and running `make html` (etc) should be familiar to folks who are already familiar with Sphinx.  The (hypothetical) Sphinx experts who are only contributing doc improvements may not want or need to set up a dev environment just to update docs.

Reply:
Those Sphinx experts can also easily figure out that their normal workflow remains unaffected, so I don't know.  Maybe I should drop that sentence.

Reply:
Dropped that paragraph in 5b39940.